backend/models/userModel.js

```diff
@@ -1,5 +1,6 @@
 const mongoose = require('mongoose');
 
+// TODO: We have to code functions for completing todolists and tasks
```

Comment:
Can't we do it with our PATCH functions?

Reply:
Okay, I get it - it's for stats :) But we can consider adding a condition to our functions instead of adding new functions. This condition, when the 'isdone' property changes from false to true, adds +1 to the statistics, and when changed from true to false, subtracts -1 from the statistics. It's just an idea, and if it turns out to be complicated, we can also add new functions

Reply:
I mean completing todolist, and adding a monster to profile